azure.go

```diff
@@ -289,3 +290,19 @@ func (b *azureAuthBackend) getAzureSettings(ctx context.Context, config *azureCo
 
 	return settings, nil
 }
+
+func ConfigurationFromName(name string) (cloud.Configuration, error) {
+	configs := map[string]cloud.Configuration{
```

Comment:
I think we are missing [AzureGermanCloud](https://developer.hashicorp.com/vault/api-docs/auth/azure#environment)

Reply:
Looks like Microsoft may have removed or missed `AzureGermanCloud` as well: https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/azcore/cloud/cloud.go

We could create one internally and add that to the list, but I wanted to make sure that Microsoft hadn't actually deprecated that entirely

Reply:
Ah, yes it is closed. See https://learn.microsoft.com/en-us/power-platform/admin/about-microsoft-cloud-germany